BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -10,7 +10,7 @@ namespace BlogTemplate._1.Pages
     public class PostModel : PageModel
     {
         private readonly BlogDataStore _dataStore;
-
+        public bool MaxedOutComments { get; set; }

```

Comment:
You could probably just set this default to `false` so that you don't have to set it in line 32.

Reply:
Thanks, just fixed it in the new commit.